src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -109,24 +109,6 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
                     name: "PSCommandNotFoundSuggestion",
                     description: "Recommend potential commands based on fuzzy search on a CommandNotFoundException"),
-                new ExperimentalFeature(
-                    name: "PSForEachObjectParallel",
-                    description: "New parameter set for ForEach-Object to run script blocks in parallel"),
-                new ExperimentalFeature(
-                    name: "PSTernaryOperator",
-                    description: "Support the ternary operator in PowerShell language"),
-                new ExperimentalFeature(
-                    name: "PSErrorView",
-                    description: "New formatting for ErrorRecord"),
-                new ExperimentalFeature(
-                    name: "PSUpdatesNotification",
```

Comment:
The code that uses this feature name is not cleaned up.

Reply:
I thought I did a search for each one, looks like I missed this case.